extra/ldr-to-hdr.sh

```diff
@@ -0,0 +1,226 @@
+      ;;
+    b) # Camera Response File
+      CONFIG_CAMRESP_PATH=$OPTARG
+      ;;
+    n) # Camera Setting File
+      # Use regex magic to parse the file for all the information we need.
+      CONFIG_FISHEYE_DIAM=$(grep -Po "(?<=diameter <- ).*" $OPTARG)
+      CONFIG_CROP_X=$(grep -Po "(?<=xleft <- ).*" $OPTARG)
+      CONFIG_CROP_Y=$(grep -Po "(?<=ydown <- ).*" $OPTARG)
+      CONFIG_VA_V=$(grep -Po "(?<=-vv ).*" $OPTARG)
+      CONFIG_VA_H=$(grep -Po "(?<=-vh ).*" $OPTARG)
+      CONFIG_OUT_X=$(grep -Po "\d*(?=x\d*)" $OPTARG)
+      CONFIG_OUT_Y=$(grep -Po "(?<=\dx)\d*" $OPTARG)
+      ;;
+      
+  esac
+done
+
+# Ensure all variable are set
+
+if [[ -z $INPUT_FILES ]]; then
+	echo "ERROR: No input files given, please give as path to all files."
+	exit 1
+fi
+if [[ -z $CONFIG_CALFAC_PATH ]]; then
+	echo "ERROR: No calibration factoring file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_VIG_PATH ]]; then
+	echo "ERROR: No vignetting config file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_FISHEYE_PATH ]]; then
+	echo "ERROR: No fisheye correction config file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_CAMRESP_PATH ]]; then
+	echo "ERROR: No camera response config file given, please give file."
+	exit 1
+fi
+if [[ -z $CONFIG_FISHEYE_DIAM || -z $CONFIG_CROP_X || -z $CONFIG_CROP_Y || -z $CONFIG_VA_V || -z $CONFIG_VA_H || -z $CONFIG_OUT_X || -z $CONFIG_OUT_Y ]]; then
+	echo "ERROR: Camera settings file did not give all information needed! Please check it."
+	exit 1
+fi
+
+# Set up the temp files, something simple.
+TEMP_FILENAME="$TEMP_FILEPATH/$(md5sum ${INPUT_FILES[0]} | awk '{ print $1 }').hdr"
+TEMP_FILENAME_ALT="$TEMP_FILEPATH/$(md5sum ${INPUT_FILES[1]} | awk '{ print $1 }').hdr"
+
+# Make sure neither file already exists
+rm $TEMP_FILENAME 2> /dev/null
+rm $TEMP_FILENAME_ALT 2> /dev/null
+
+# One last check, let's make sure that the Radiance suite and hdrgen are in PATH.
+
+# hdrgen
+if ! $(which hdrgen 2>&1 > /dev/null); then
+	echo "ERROR: Missing PATH to hdrgen, please add to your PATH or set with -h flag."
+	exit 1
+fi;
+
+# pcomb from Radiance
+if ! $(which pcomb 2>&1 > /dev/null); then
+	echo "ERROR: Missing PATH to Radiance utilities, please add to your PATH or set with -r flag."
+	exit 1
+fi;
+# We have Radiance and hdrgen available!
+
+# With all of the images selected, generate the HDR image.
+hdrgen ${INPUT_FILES[@]} -o $TEMP_FILENAME -r $CONFIG_CAMRESP_PATH -a -e -f -g
+
+echo "Compiled the initial HDR image, now processing..."
+
+# Exposure Nullification
+echo "Nullifying Exposures..."
+
+ra_xyze -r -o $TEMP_FILENAME > $TEMP_FILENAME_ALT
+
+# Image Cropping
+echo "Cropping Image..."
+
+pcompos -x $CONFIG_FISHEYE_DIAM -y $CONFIG_FISHEYE_DIAM $TEMP_FILENAME_ALT -$CONFIG_CROP_X -$CONFIG_CROP_Y > $TEMP_FILENAME
+
+# Vignetting Correction
+echo "Correcting the vignetting..."
+
+pcomb -f $CONFIG_VIG_PATH -e "diameter=$CONFIG_FISHEYE_DIAM" $TEMP_FILENAME > $TEMP_FILENAME_ALT
```

Comment:
I think this step is out of order with the tutorial version (which is the one we're following). This document has a list of the commands in the correct order: [workflow description](https://docs.google.com/document/d/1tT4zebzbG_tH8EO-0-ZmEQjnrIsFu-yoXjZ5pYDEnx0/edit#heading=h.ukmubzu9i908).

Additionally, I believe it should not have the `-e` flag.

Reply:
Correct! No, `-e` flag :)